src/Dialogs.vala

```diff
@@ -154,7 +154,7 @@ namespace Gala {
                 title = _("Application is not responding");
             }
 
-            body = _("You may choose to wait a short while for it to continue or force the application quit entirely.");
+            body = _("You may choose to wait a short while for it to continue or force the application to quit entirely.");
```

Comment:
I think it better to use the specific noun first and "it" second.

-> "You may choose to wait a short while for the application to continue, or force it to quit entirely."

Reply:
Agreed! Done